src/editor/Model/BufferMap.re

```diff
@@ -23,6 +23,14 @@ let map = Buffers.map;
 let update = Buffers.update;
 let remove = Buffers.remove;
 
+let getBuffers = (buffers: t) =>
```

Comment:
This name is a bit confusing - it returns a list of buffer metadatas, right? I would expect that `getBuffer` would give me a single buffer, whereas `getBuffers` would return a list of buffers.

Reply:
this could be `getBufferMetadata` if it seems too confusing?